### PR TITLE
Amjith/release 1.6.0

### PR DESCRIPTION
**Pull request by amjith**
This is in preparation for 1.6.0 release. 

Can the @dbcli/mycli-core  team please review the change log to make sure I didn't skip something? 

I'll merge this in once I get a :+1: from a majority of core devs.

### CODE REVIEW

**Comment by tsroten** [comment]
@amjith Thanks for preparing this release. I think the changelog is missing #219 and #226.

File: changelog.md [hunk 1]
```diff
@@ -1,3 +1,35 @@
+1.6.0:
+======
+
+Features:
+---------
+
+* Change continutation prompt for multi-line mode to match default mysql.
```
Comment (matheus-iclinic) [inline changelog.md]: @amjith Here you mean "continuation", right? :smile:

Comment (mdsrosa) [inline changelog.md]: @amjith Here you mean "continuation", right? :smile:

**Comment by amjith** [comment]
I'm sorry I missed the two features @tsroten, it was an oversight. I've fixed it now. 

The typo is also fixed @mdsrosa. 

Can you take another look?

**Comment by tsroten** [comment]
@amjith Looks good to me!

**Comment by mdsrosa** [comment]
@amjith Looks good to me too!

**Comment by amjith** [comment]
I'm going to merge this and do a release. We'll release the stdin feature in the next release. I'm planning to do more frequent releases (once every couple of weeks).

**Pull request merged**